include/vcpkg/base/messages.h

```diff
@@ -1246,6 +1307,7 @@ namespace vcpkg
                     "",
                     "Package {spec} is installed, but dependency {package_name} is not.");
     DECLARE_MESSAGE(MissingExtension, (msg::extension), "", "Missing '{extension}' extension.");
+    DECLARE_MESSAGE(MissingNDKVersion, (msg::path), "", "NDK version missing: {path}");
```

Comment:
I think this error is unactionable but it's what the old code did so no change required.